Source/CsDebugScript.Engine/Engine/Utility/DictionaryCache.cs

```diff
@@ -62,6 +67,22 @@ public int Count
         }
 
         /// <summary>
+        /// Gets the Cached values.
+        /// </summary>
+        IEnumerable ICacheCollection.ValuesRaw { get { return Values; } }
+
+        /// <summary>
+        /// Gets a value indicating whether there are cached values.
+        /// </summary>
+        /// <value>
+        ///   <c>true</c> if cached; otherwise, <c>false</c>.
+        /// </value>
+        bool ICache.Cached { get { return isCached; } }
+
+        object ICache.ValueRaw { get { return values.Values; }
```

Comment:
Also, comment is missing.

---
In reply to: [70602160](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70602160) [](ancestors = 70602160)